dpgen/auto_test/Elastic.py

```diff
@@ -221,7 +221,12 @@ def post_process(self, task_list):
                 kspacing = incar.get("KSPACING")
                 kgamma = incar.get("KGAMMA", False)
                 ret = vasp.make_kspacing_kpoints(poscar_start, kspacing, kgamma)
-                kp = Kpoints.from_string(ret)
+                pymgv = "old"
```

Comment:
I would not recommend introducing the pymgv variable, but rather use `try`...`except` everywhere various interfaces are introduced in different pymatgen versions.  suppose pymatgen introduce a third variety of `from_str`, we need to add a new value other than the "old" "new" to pymgv, "new2"? which is anyway not a nice name. 

Another option is that we provide an interface for calling with different versions of pymatgen, then the `try`...`except` only appear once in the code.

Reply:
Thanks for your recommendation! The extra variable `pymgv` is fully replaced by  `try`...`except` now. (I have tried to get the version of Pymatgen directly, but it seems that Pymatgen had no member like `__version__` at all.)